src/components/ActiveFilters.vue

```diff
@@ -44,25 +44,40 @@ export default Vue.extend({
   },
   watch: {
     value: {
-      handler (newValue) {
-        this.buildActiveValues(newValue)
+      handler (newActiveValues, oldActiveValues) {
+        if (newActiveValues !== oldActiveValues && (this.filters && this.filters.length > 0)) { // We need to have filterdefinitions
```

Comment:
Filters are a required component prop. There is no need to check for this.filters right?
Also, this line is self-explanatory.